src/main.rs

```diff
@@ -87,17 +87,26 @@ enum Commands {
         #[arg(short = 'n', long)]
         name: String,
         /// The number of safenode processes to run on each VM.
-        #[clap(long, default_value = "40")]
+        #[clap(long, default_value_t = 40)]
         node_count: u16,
         /// The cloud provider to deploy to.
         ///
         /// Valid values are "aws" or "digital-ocean".
         #[clap(long, default_value_t = CloudProvider::DigitalOcean, value_parser = parse_provider, verbatim_doc_comment)]
         provider: CloudProvider,
+        /// Protocol version is used to partition the network and would not allow nodes with different protocol versions
+        /// to join us.
+        ///
+        /// If set to 'restricted' then the branch name is used as the protocol version string else the passed in value
+        /// is used as the version string.
+        ///
+        /// The protocol version argument is mutually exclusive with the --safenode-version argument.
+        #[arg(long)]
+        protocol_version: Option<String>,
```

Comment:
Sorry, just a tiny little pedantic thing: `protocol_version` should come before `provider` in alphabetical ordering.